db/version_set.cc

```diff
@@ -4137,7 +4137,8 @@ Status VersionSet::ProcessManifestWrites(
                      "\n",
                      manifest_file_number_, pending_manifest_file_number_);
       env_->DeleteFile(
-          DescriptorFileName(dbname_, pending_manifest_file_number_));
+              DescriptorFileName(dbname_, pending_manifest_file_number_))
+          .PermitUncheckedError();
```

Comment:
should it log a error if a file failed to delete?

Reply:
Good catch. Will update.